trillian/docs/ManualDeployment.md

```diff
@@ -360,6 +350,22 @@ Signature: Hash=SHA256 Sign=ECDSA
 Value=3045022100df855f0fd097a45070e2eb244c7cb63effda942f2d30308e3b84a72e1d16118b0220038e55f142501402cf03790b3997081f82ffe47f2d3f3b667e1c484aecf40a33
 ```
 
+**Cross-check**: Once the CTFE is configured and running, opening
+`http://localhost:<port>/<prefix>/ct/v1/get-roots` shows the configured roots.
+Alternatively, the `ctclient` command-line tool shows the same information in a
+more friendly way:
+
+```bash
+go run github.com/google/certificate-transparency-go/client/ctclient@master getroots --log_uri http://localhost:6966/aramis
```

Comment:
The command `getroots` should be `get-roots`. We can make use of the `TRILLIAN_LOG_SERVER_RPC_ENDPOINT` for the `--log_uri`. I am not sure if the log path prefix is correct.

Reply:
Oh right, 6966 is already in the previous command.

Reply:
`get-roots`: as yes, you're right - I just moved this block more or less as-is, thanks!
`log-uri`: sadly not, this is the URI of the `ctfe` rather than the `logserver`.
`prefix`: this is entirely dependent on the ctfe config file the reader creates, so is only really an example of the command (I've added "e.g." before the command to hint at that)